Rondo/UI/Messages/MessagesViewController.swift

```diff
@@ -28,9 +28,32 @@ class MessagesViewController: FormViewController {
             }
         }
         
+        SwitchRow.defaultCellSetup = { cell, row in
+            cell.selectionStyle = .none
+            row.onChange { [weak self] (row) in
+                guard let self = self, let value = row.value else { return }
+                self.setUNUserNotificationCenterDelegate(on: value)
+            }
+            row.onCellSelection { (cell, row) in
+                guard let value = row.value else { return }
+                row.value = !value
+                cell.update()
+            }
+        }
+        
         addSegmentedControl()
         buildIAM()
     }
+    
+    private func setUNUserNotificationCenterDelegate(on: Bool) {
+        if on {
+            UNUserNotificationCenter.current().delegate = UIApplication.shared.appDelegate
+        } else {
+            UNUserNotificationCenter.current().delegate = nil
+        }
+        UserDefaults.standard.set(on, forKey: "useUNUserNotificationCenterDelegate")
```

Comment:
Move this to UserDefaultsExtension.swift.
UserDefaults.standard.synchronize() shouldnt be used.